eox_tenant/monkey_patch/monkey_patch_proxys.py

```diff
@@ -0,0 +1,160 @@
+"""
+Monkey patch proxys that allows to override the platform models.
```

Comment:
the plural of proxy is proxies. We might have to replace this all over

Reply:
Also I suggest we change the name of this file from moneky_patch/monkey_patch_proxys.py to proxies/models.py

Reply:
@felipemontoya  and if we need to override another thing that is not a proxy ??